src/Symfony/Component/Form/Tests/Extension/HttpFoundation/EventListener/LegacyBindRequestListenerTest.php

```diff
@@ -15,14 +15,13 @@
 use Symfony\Component\Form\Form;
 use Symfony\Component\Form\FormConfigBuilder;
 use Symfony\Component\Form\FormEvent;
-use Symfony\Component\Form\Test\DeprecationErrorHandler;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-class BindRequestListenerTest extends \PHPUnit_Framework_TestCase
+class LegacyBindRequestListenerTest extends \PHPUnit_Framework_TestCase
```

Comment:
I plan to enhance the deprecation summary generator to take this prefix into account, so: yes :)